uint32 id
string name
float64 positionX
float64 positionY
float64 positionZ
float64 orientationRoll
float64 orientationPitch
float64 orientationYaw
uint32[] inArea

